src/theme/defaultTheme.js

```diff
@@ -69,9 +69,11 @@ const colors = {
 export const chartColors = {
   lightblue: '#00ACEB',
   darkblue: '#2b388f',
-  purple: '#A6278F',
+  lightpurple: '#A6278F',
   orange: '#F79122',
   red: '#dd1f2a',
+  blue: '#1f9bb6',
+  purple: '#e3429b',
```

Comment:
@nyanofthemoon sorry if I wasn't making sense earlier, this is what I meant. Does it make sense?

<img width="1068" alt="screen shot 2019-01-31 at 1 21 52 pm" src="https://user-images.githubusercontent.com/11821106/52076926-ce9b9600-255d-11e9-8d7a-40a13e72aa8d.png">

Reply:
Was changed in previous, this change puts it back. Thanks @nyanofthemoon for the clarification! :)

Reply:
@nyanofthemoon why rename? `theme.chartColors.purple` is used in UserDashboard charts.